plugins/openstack/keystone/check_keystone-api.sh

```diff
@@ -79,11 +88,11 @@ if [ -z "$TOKEN" ]; then
     echo "Unable to get a token"
     exit $STATE_CRITICAL
 else
-    if [ $TIME -gt 10 ]; then
-        echo "Get a token after 10 seconds, it's too long."
+    if [ $TIME -gt $TIMEOUT ]; then
+        echo "Got a token after $TIMEOUT seconds. That is too long."
```

Comment:
How about "It took too long to retrieve a token. Time taken: $TIMEOUT seconds". There are way too many reasons why it took so long to retrieve the token. It could be that the server was under high load, the network is saturated, too many users submitting API requests, etc., etc.